maker/src/orderbook_ws.rs

```diff
@@ -87,35 +95,61 @@ impl Client {
 async fn process_message(
     msg: String,
     position_manager: &xtra::Address<position::Manager>,
+    maker_trader_id: &PublicKey,
 ) -> Result<()> {
     tracing::trace!(%msg, "New message from orderbook");
 
     let msg = serde_json::from_str::<OrderbookMsg>(&msg).context("Deserialization failed")?;
 
     match msg {
+        OrderbookMsg::AllOrders(orders) => {
```

Comment:
🚨You will only received non-expired orders here. You won't get consistently the taken orders here over time as the makers orders expire after 60 seconds. (unless configured differently)

Reply:
Thanks for the tip. The name definitely does not convey this.

I'm going to add an API to the WebSocket to get all the matched/taken orders for a trader.

Reply:
It turns out none of the current orderbook messages work well for this purpose, so I just ended up adding a new one and letting the maker ask for it via a new orderbook request.

It would be much nicer if the orderbook could drive these updates instead, but it's not easy to implement, specially since we want to avoid adding maker-specific logic to the orderbook/coordinator.